src/open_dread_rando/pickups/split_pickups.py

```diff
@@ -28,29 +33,28 @@ class MissileInput(Enum):
     L = 1
 
 
```

Comment:
this whole class should just be replaced with ActorDefFunc outright

Reply:
I didn't want to make big changes, just fix

Reply:
`ActorDefFunc` was literally copy pasted from this and was designed to be a drop-in replacement